pom.xml

```diff
@@ -16,45 +16,45 @@
     limitations under the License.
 
 -->
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
-    <groupId>com.forgerock.securebanking.uk</groupId>
-    <artifactId>securebanking-openbanking-uk</artifactId>
+    <groupId>com.forgerock.sapi.gateway</groupId>
+    <artifactId>secure-api-gateway-ob-uk-common</artifactId>
     <packaging>pom</packaging>
-    <version>1.1.9-SNAPSHOT</version>
-    <name>Secure Banking access Toolkit library for Open Banking UK specs</name>
-    <url>https://github.com/SecureBankingAccessToolkit/securebanking-openbanking-uk</url>
+    <version>1.0.0-SNAPSHOT</version>
+    <name>Secure API Gateway common and shared libraries for Open Banking UK specs</name>
```

Comment:
I thought we agreed to use a 0.9.0 to start until we get everything building, then we do the first release using the release process to 1.0.0?

Reply:
For the release candidate process I will pass the release version as parameter

Reply:
although we can use 0.9.0-SNAPSHOT as development iteration currently

Reply:
Yes, so that you can the test the release process to do a release of 0.9.1 for example... 

When all the release process is in place and working we can then do a release of everything at 1.0.0!

Reply:
done! we will use the developer version 0.9.0-SNAPSHOT